xml_converter/generators/code_generator.py

```diff
@@ -296,6 +296,9 @@ class AttributeVariable:
     uses_file_path: bool = False
 
 
+DEFAULT_ARGUMENTS: Final[List[str]] = ["attribute", "errors"]
```

Comment:
This global needs a more descriptive name to make it clear what it is for. DEFAULT_ARGUMENTS could be "arguments for this script" or something. EG: "ATTRIBUTE_PARSER_DEFAULT_ARGUMENTS" or something. Go with your gut here, naming is hard.